spec/index.html

```diff
@@ -363,6 +363,39 @@ <h3>Triple Terms and Reification</h3>
 
     <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
   </section>
+  
+  <section id="section-version-announcement">
+    <h3>RDF Version Announcement</h3>
+	
+    <p>To allow RDF parsers to error or warn about unsupported RDF versions as early as possible,
+		RDF serialization formats are expected to allow a version to be specified,
+		via either the HTTP Content-Type header,
+		a version announcement in the format-specific syntax,
+		or both.</p>
+
+	<p>If the version is announced in both the Content-Type header and in the syntax, they are expected to be equal.</p>
```

Comment:
Implementations should be advised on what to do in the case of a discrepancy between the two values:

```suggestion
	<p>When the version is indicated both in the <code>Content-Type</code> header and in the syntax,
	  they are expected to be the same.
	  If they differ,
	  parsers use the <code>Content-Type</code> version and emit a warning about the mismatch.</p>
```

Reply:
The alternatives would be for parsers to:
1. pick the value from the syntax,
2. ignore, or
3. error.

My feeling is that parsers should this, given that multiple `VERSION` directives could also be defined in Turtle, with different values.

In any case, `1.2` is the only allowed value at the moment, so perhaps this sentence is even not required at this stage.

Reply:
This kind of situation exists already on the web: charset can be provided by both HTML and HTTP headers, and I believe that there are known good practices on how to prioritize (although I didn't have time to look them up).

Reply:
We need to be clear that having parsers emit a warning is not a requirement, but an opportunity. At this point, any use that is not "1.2" can be considered an exception and could generate a warning or an error. The situation would be similar if we did allow a further range of values, and they didn't agree (e.g., media-type parameter is "1.2", but `VERSION` is "1.2-dev").

Regarding the charset discrepancy, I saw a reference to [5.2.2 of HTML401](https://www.w3.org/TR/html401/charset.html#h-5.2.2):

> To sum up, conforming user agents must observe the following priorities when determining a document's character encoding (from highest priority to lowest):

> 1. An HTTP "charset" parameter in a "Content-Type" field.
> 2. A [META](https://www.w3.org/TR/html401/struct/global.html#edef-META) declaration with "http-equiv" set to "Content-Type" and a value set for "charset".
> 3. The [charset](https://www.w3.org/TR/html401/struct/links.html#adef-charset) attribute set on an element that designates an external resource.

This suggests that the version specified in the media-type be used in preference to an inline `VERSION`. It does not say what to do if there is a discrepancy, but I think this can be left up to each parser.